atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTopologyValidatorTest.java

```diff
@@ -97,12 +96,12 @@ public void retriesThrowsAndMarksFailureWhenNeverResolves() {
         EntryStream.of(allHosts)
                 .values()
                 .forEach(container -> setHostIds(Set.of(container), HostIdResult.success(Set.of(uuidIterator.next()))));
-        assertThatThrownBy(() -> validator.getNewHostsWithInconsistentTopologiesAndRetry(
+        assertThat(validator.getNewHostsWithInconsistentTopologiesAndRetry(
```

Comment:
nit: test name is wrong now since we're not throwing! Probably also worth asserting that when it never resolves, it returns the inconsistent hosts (rather than some arbitrary set of hosts!)